transcom/mymove
# B-21667 I-13574 INT

Upstreams:
https://github.com/transcom/mymove/pull/14176
https://github.com/transcom/mymove/pull/14188
https://github.com/transcom/mymove/pull/14246

## [Agility issue](https://www13.v1host.com/USTRANSCOM38/Issue.mvc/Summary?oidToken=Issue%3A1036171)
## [Agility ticket](https://www13.v1host.com/USTRANSCOM38/assetdetail.v1?number=B-21667)

## Summary

Needed to update ubAllowance to unaccompaniedBaggageAllowance, after changing that variable name for the prime API as part of [B-21487](https://www13.v1host.com/USTRANSCOM38/assetdetail.v1?number=B-21487), which I have merged into my B-21667 branch.

How to test

1. Login as a customer, creating a UB shipment with the UB FF turned on
2. Log in as an SC and view the Allowances section for the move and check that you have an Unaccompanied baggage allowance
![image](https://github.com/user-attachments/assets/3f889970-ea61-47f0-b337-84492e768e1a)
3. Click on the Edit shipment button
4. Verify that the UB Weight Allowance shown at the top of the Edit Shipment Details page matches the number on the Allowances section
![image](https://github.com/user-attachments/assets/32ded990-bc7a-493c-af65-f3da674f4381)
5. Do this for both SC and TOO
